Kahi_doaj_sources/kahi_doaj_sources/Kahi_doaj_sources.py

```diff
@@ -0,0 +1,206 @@
+                    {"lang": "en", "name": reg["title"], "source":"doaj"}]
+                entry["keywords"] = reg["keywords"]
+                entry["languages"] = reg["language"]
+                entry["publisher"] = {"country_code": reg["publisher"]
+                                      ["country"], "name": reg["publisher"]["name"], "id": ""}
+                entry["open_access_start_year"] = reg["oa_start"] if "oa_start" in reg.keys(
+                ) else None
+                entry["external_urls"] = [
+                    {"source": ref, "url": url} for ref, url in reg["ref"].items()]
+                entry["review_process"] = reg["editorial"]["review_process"]
+                entry["plagiarism_detection"] = reg["plagiarism"]["detection"]
+                entry["publication_time_weeks"] = reg["publication_time_weeks"]
+                entry["copyright"] = reg["copyright"]
+                entry["licenses"] = reg["license"]
+
+                if "apc" in reg.keys():
+                    if reg["apc"]["has_apc"]:
+                        entry["apc"] = {"charges": reg["apc"]["max"][-1]["price"],
+                                        "currency": reg["apc"]["max"][-1]["currency"]}
+
+                subjects_source = {}
+                if "subject" in reg.keys():
+                    if reg["subject"]:
+                        for sub in reg["subject"]:
+                            sub_entry = {
+                                "id": "",
+                                "name": sub["term"],
+                                "external_ids": [{"source": sub["scheme"], "id":sub["code"]}]
+                            }
+                            if sub["scheme"] in subjects_source.keys():
+                                subjects_source[sub["scheme"]].append(
+                                    sub_entry)
+                            else:
+                                subjects_source[sub["scheme"]] = [sub_entry]
+                for source, subs in subjects_source.items():
+                    entry["subjects"].append({
+                        "source": source,
+                        "subjects": subs
+                    })
+
+                if "eissn" in reg.keys():
+                    entry["external_ids"].append(
+                        {"source": "eissn", "id": reg["eissn"]})
+                if "pissn" in reg.keys():
+                    entry["external_ids"].append(
+                        {"source": "pissn", "id": reg["pissn"]})
+
+                entry["waiver"] = reg["waiver"]
+
+                response = self.collection.insert_one(entry)
+                for ext in entry["external_ids"]:
+                    self.already_in_db.append(ext["id"])
+                if verbose > 4:
+                    print("Inserted: ", response.inserted_id)
+                if verbose > 0:
+                    print("Total inserted: ", len(self.already_in_db))
+
+                delta = dt.datetime.now()-old
```

Comment:
this is the error in the actions
```suggestion
                delta = dt.datetime.now() - old
```